manifests/kustomize/env/platform-agnostic-tekton/kustomization.yaml

```diff
@@ -8,6 +8,7 @@ resources:
 - ../../third-party/tekton-custom-task
 - ../../third-party/minio/base
 - ../../third-party/mysql/base
+- ../../third-party/kfp-csi-s3
```

Comment:
Do we need kfp-csi-s3 for v2? This was mainly meant for copying big artifacts into s3. Do we have any pvc implementation that uses this csi driver by default in v2?

Reply:
no built-in usage for that. but you can create/user volume in your pipeline by using kfp-kubernetes sdk

Reply:
I guess we can put the manifest in, but not have it in the kustomization.yaml

Reply:
We shouldn't include it by default then, in iks the cos driver has to install separately. If nothing is using it by default we should make users to opt-in if they want to install kfp-csi-s3.

Reply:
One of the main reason is that cos driver only support iam auth, we can't use default minio auth on cos driver.